docs/reference/api-react-search-provider.md

```diff
@@ -12,12 +12,12 @@ It looks like this:
 
 ```jsx
 import { SearchProvider, SearchBox } from "@elastic/react-search-ui";
-import AppSearchAPIConnector from "@elastic/search-ui-app-search-connector";
+import ElasticSearchAPIConnector from "@elastic/search-ui-elasticsearch-connector";
 
-const connector = new AppSearchAPIConnector({
-  searchKey: "search-371auk61r2bwqtdzocdgutmg",
-  engineName: "search-ui-examples",
-  endpointBase: "http://my-app-search-host:3002"
+const connector = new ElasticSearchAPIConnector({
+  host: "https://search-ui-sandbox.es.us-central1.gcp.cloud.es.io:9243",
+  index: "national-parks",
+  apiKey: "SlUzdWE0QUJmN3VmYVF2Q0F6c0I6TklyWHFIZ3lTbHF6Yzc2eEtyeWFNdw=="
```

Comment:
any issues with this being in a public repo?

Reply:
No, I took this creds from examples